src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -129,7 +129,11 @@ def load_arguments(self, _):
         option_list=['--enable-arm-private-network-access'],
         arg_type=get_three_state_flag(),
         help="Enable access to the App Configuration store via ARM Private Link if resource is restricted to private network access. Requires Pass-through ARM authentication mode."
```

Comment:
Need to update help message of  snapshot_filter_arg_type to include tags filters